File: joao pedro/Problema0387/Problema0387.java
import java.util.Scanner;

public class Problema0387 {
    public static void main(String[] args) {
        Scanner input=new Scanner(System.in);
        int [][]matriz=new int[3][3];
        System.out.println("Digite valores inteiros: ");
        for(int i=0;i<3;i++){
            for(int j=0;j<3;j++){
                matriz[i][j]=Integer.valueOf(input.nextLine());
            }
        
        }
        int x = matriz.length;
        int y = matriz[0].length;
        int[][] rotacionado = new int[y][x];
        for (int i=0; i<y; i++) {
        for (int j=0; j<x; j++) {
            rotacionado[i][j] = matriz[x - j - 1][i];
    }}
    for(int i=0;i<3;i++){
        System.out.println("");
        for(int j=0;j<3;j++){
            System.out.print(rotacionado[i][j]);
        }
    }
}}
    

File: joao pedro/Problema0387/Problema0387.py
matriz = [[0] * (3) for _ in range(3)]
print("Digite valores inteiros: ")
i = 0
while (i < 3):
    j = 0
    while (j < 3):
        matriz[i][j] = int(input())
        j += 1
    i += 1
x = len(matriz)
y = len(matriz[0])
rotacionado = [[0] * (x) for _ in range(y)]
i = 0
while (i < y):
    j = 0
    while (j < x):
        rotacionado[i][j] = matriz[x - j - 1][i]
        j += 1
    i += 1
    
i = 0
while (i < 3):
    print("")
    j = 0
    while (j < 3):
        print(rotacionado[i][j], end="")
        j += 1
    i += 1
